docs/kubernetes/KUBERNETES.md

```diff
@@ -25,22 +25,24 @@ kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/relea
 
 4. Customization:
 
-    4.1 In `ConfigMap.yml`: You need to change the values according to your cluster. You can set any of the variables from the [list here](docs/ENV.md) to trailor setup to your requirements.
+    4.1 In `.env.`: Change the domain name to your domain.
 
-    4.2 In `Service.yml`: Change the domain name to your domain and the set the external IP addresses for `openvpn` & `freeradius` services.
+    4.2 In `ConfigMap.yml`: You need to change the values according to your cluster. You can set any of the variables from the [list here](docs/ENV.md) to trailor setup to your requirements.
 
-    4.3 In `Ingress.yml`: Change the domain name to your domain and the set the load-balaner IP address.
+    4.3 In `Service.yml`: Set the external IP addresses for `openvpn` & `freeradius` services.
 
-    4.4 In `CertManager.yml`: Change the email address & the domain name to your domains.
+    4.4 In `Ingress.yml`: Set the load-balaner IP address.
```

Comment:
Ah, that's my bad, remove this step, on bare-metal this is not required! 
But you just installed on bare-metal, didn't you face this? :smile: